### ISSUE TITLE
Include instructions on how to switch from Mojarra 3 to 4 (Jakarta Faces)

### ISSUE THREAD

**Issue by Master-Code-Programmer** [joinfaces/joinfaces#1691]
JoinFaces works great, but its dependency Mojarra 3 has bugs which are fixed in Mojarra 4.
And Mojarra 3 is still the default for JoinFaces, and it's not obvious how to tell JoinFaces that you want to use Version 4.

I just would like to suggest to add to the documentation how to do it.

My solution for Gradle Kotlin was:
```kotlin
implementation("org.joinfaces:jsf-spring-boot-starter:5.0.4") {
  exclude(
    "org.joinfaces",
    module = "mojarra3-spring-boot-starter"
  )
}
implementation("org.joinfaces:mojarra4-spring-boot-starter:5.0.4")
```
This is far from obvious. There is already a documentation on how to swap Mojarra with MyFaces,
under https://docs.joinfaces.org/5.0.0-m4/reference/#_not_using_tomcat_or_mojarra.
Maybe that's the right place to also tell the user how to switch Mojarra versions?

Also I'm wondering why Mojarra 3 is still the default. When we switched from Mojarra 3 to 4 we got rid of 102 exceptions, which were coming up in the logs on debug-log-level. And we had zero problems in any other software, with the switch from Mojarra 3 to 4.

**Comment by larsgrefer** [comment]
TL;DR: Use `org.joinfaces:faces-spring-boot-starter` instead of `org.joinfaces:jsf-spring-boot-starter`.

`org.joinfaces:jsf-spring-boot-starter` is the "old" starter from JoinFaces 4.x which used to pull Mojarra 2.3.
With JoinFaces 5.x this starter now pulls Mojarra 3 in order to provide a smoother upgrade path (Mojarra 3 and 2.3 are basically the same apart from the javax to jakarta rename)

`org.joinfaces:faces-spring-boot-starter` was added with JoinFaces 5 and pulls Mojarra 4.

**Comment by Master-Code-Programmer** [comment]
> TL;DR: Use `org.joinfaces:faces-spring-boot-starter` instead of `org.joinfaces:jsf-spring-boot-starter`. ...

Thanks, the changes from https://github.com/joinfaces/joinfaces/commit/e507def910e6e77a84347a69d205166c91c4fe46 look great.

**Comment by Master-Code-Programmer** [comment]
I don't have the time right now to _really_ test this, but it reads great so far, so I close this issue as solved.
